data/iso-codes/data_iso_3166-2.json

```diff
@@ -13571,6 +13571,66 @@
       "name": "Luxembourg",
       "type": "District"
     },
```

Comment:
The only think I'm not sure is about leaving the old district in the JSON file

Reply:
makes sense, changed